event/event_cache_test.go

```diff
@@ -178,7 +178,7 @@ func TestSubManualClose(t *testing.T) {
 // Test that the system doesn't fail under high pressure.
 func TestSubFlooding(t *testing.T) {
 	NUM_SUBS := 100
-	// Keep the reaper out of evts.
+	// Keep the reaper out of this.
```

Comment:
i could have sworn a reviewed a logging PR very recently where `s/this/evts` ... any reason you're going back the other way?